packages/editor/src/client/routes/commentList.tsx

```diff
@@ -348,23 +364,54 @@ export function CommentList() {
               <DescriptionListItem label={t('comments.panels.updatedAt')}>
                 {currentComment?.modifiedAt && new Date(currentComment.modifiedAt).toDateString()}
               </DescriptionListItem>
-              <DescriptionListItem label={t('comments.panels.revisions')}>
-                <Timeline align="left">
-                  {currentComment?.revisions?.length
-                    ? currentComment?.revisions?.map(({text, createdAt}, i) => (
-                        <Timeline.Item key={i}>
-                          <div>{new Date(createdAt).toLocaleString()}</div>
+
+              {parentComment?.comment && (
+                <>
+                  <DescriptionListItem label={t('comments.panels.parent')}>
+                    <Panel bordered style={{marginRight: 40, fontStyle: 'italic', color: 'gray'}}>
+                      {parentComment?.comment?.revisions[length - 1].text.length ? (
+                        <>
+                          <div>
+                            {new Date(
+                              parentComment?.comment?.revisions[length - 1].createdAt
+                            ).toLocaleString()}
+                          </div>
                           <RichTextBlock
-                            disabled
                             displayOnly
+                            displayOneLine
+                            disabled
                             // TODO: remove this
                             onChange={console.log}
```

Comment:
you can remove the TODO comment and replace console.log with `/* do nothing */`

Reply:
@roosnic1, ts won't let me assign an empty value here.